### PR TITLE
Disable prepublish script in test.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

File: scripts/test_emscripten.sh [hunk 1]
```diff
@@ -36,6 +36,10 @@ DIR=$(mktemp -d)
     echo "Preparing solc-js..."
     git clone --depth 1 https://github.com/ethereum/solc-js "$DIR"
     cd "$DIR"
+    # disable "prepublish" script which downloads the latest version
+    # (we will replace it anyway and it is often incorrectly cached
+    # on travis)
+    npm set script.prepublish ''
```
Comment (axic) [inline scripts/test_emscripten.sh]: What is this magic? Cannot find any documentation about it.

**Comment by chriseth** [comment]
Better now?

**Comment by axic** [comment]
@chriseth was fine with the other one too just couldn't figure out the documentation

Apparently `set` is an alias for `config set`:
```
$ npm config --help
npm config set <key> <value>
npm config get [<key>]
npm config delete <key>
npm config list
npm config edit
npm set <key> <value>
npm get [<key>]

alias: c
```

**Comment by chriseth** [comment]
Sure :-) So can you please approve?

**Pull request merged**